src/RemoteTech/API/API.cs

```diff
@@ -93,37 +102,33 @@ public static double GetSignalDelayToSatellite(Guid a, Guid b)
             return delayBetween;
         }
 
-        public static void ReceiveData(ConfigNode ExternalData) //exposed method called by other mods, passing a ConfigNode to RemoteTech
+        public static void ReceiveData(ConfigNode externalData) //exposed method called by other mods, passing a ConfigNode to RemoteTech
         {
-            if (ExternalData != null) //check we were actually passed a config node
```

Comment:
The rest of this file is a refactor of the current method with no material changes.